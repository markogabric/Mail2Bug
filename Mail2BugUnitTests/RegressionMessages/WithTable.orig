<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="Generator" content="Microsoft Exchange Server">
<!-- converted from rtf -->
<style><!-- .EmailQuote { margin-left: 1pt; padding-left: 4pt; border-left: #800000 2px solid; } --></style>
</head>
<body>
<font face="Calibri" size="2"><span style="font-size:11pt;">
<div>Hello</div>
<div><font face="Arial (Hebrew)">מה נשמע</font></div>
<table width="798" style="width:478.8pt;">
<col width="266" style="width:159.6pt;">
<col width="266" style="width:159.6pt;">
<col width="266" style="width:159.6pt;">
<tr>
<td>A</td>
<td>B</td>
<td>C</td>
</tr>
<tr>
<td>D</td>
<td>E</td>
<td>F</td>
</tr>
<tr>
<td>G</td>
<td>H</td>
<td>I</td>
</tr>
</table>
<div>&nbsp;</div>
<div>&nbsp;</div>
<div>&nbsp;</div>
<div>Thanks,</div>
<div>Guy</div>
<div>&nbsp;</div>
<div>&nbsp;</div>
</span></font>
</body>
</html>
