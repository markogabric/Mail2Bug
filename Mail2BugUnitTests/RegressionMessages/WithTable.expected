Hello
מה נשמע
A
B
C
D
E
F
G
H
I

Thanks,
Guy

 